packages/react-resizable-panels/src/PanelResizeHandle.test.tsx

```diff
@@ -1,14 +1,24 @@
 import { Root, createRoot } from "react-dom/client";
 import { act } from "react-dom/test-utils";
-import type { PanelResizeHandleProps } from "react-resizable-panels";
-import { Panel, PanelGroup, PanelResizeHandle } from ".";
+import {
+  Panel,
+  PanelGroup,
+  PanelResizeHandle,
+  type PanelResizeHandleProps,
+} from ".";
```

Comment:
I've updated the way `PanelResizeHandleProps` gets imported because the original way doesn't seem to work for me, even after a clean checkout. I can revert this if necessary.

Reply:
That fixes it, thanks.